tests/JustEat.StatsD.Tests/statsdconfig.js

```diff
@@ -1,4 +1,5 @@
 {
+  keyNameSanitize: false,
```

Comment:
If this is needed to make tags work in general, we'd probably want this documented somewhere in the README.

Which reminds me, the README would also need to be updated to mention the tagging capabilities.

Reply:
Some guidance in the need to use `keyNameSanitize` if using tags would still be useful in the README.

Reply:
I think it should be always false since the tests cover tags and the StatsD server is mocked. In real life this flag does not exist, for instance when running on Graphite.

Reply:
It's not a mock, it's the real thing:

https://github.com/justeat/JustEat.StatsD/blob/34f5bf6bba124cd565c52b28e8679fbf941e1e6d/.travis.yml#L27-L29